src/Umbraco.Core/Services/PackagingService.cs

```diff
@@ -49,7 +49,7 @@ public class PackagingService : IPackagingService
         private Dictionary<string, IContentType> _importedContentTypes;
         private IPackageInstallation _packageInstallation;
         private readonly IUserService _userService;
-
+        private static readonly HttpClient HttpClient = new HttpClient();

```

Comment:
Since this one is only used in the `FetchPackageFile` method, we should only initialize a new one in that method, that way it only gets created if we really need it. So like you did elsewhere, check if it's null and then create it.